src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -219,14 +221,14 @@ private static LibraryDependency ToPackageLibraryDependency(PackageReference ref
             return dependency;
         }
 
-        private static string GetReferenceMetadataValue(PackageReference reference, string metadataElement)
+        private static string GetReferenceMetadataValue(PackageReference reference, string metadataElement, string defaultValue = "")
```

Comment:
`string.Empty` for default value

Reply:
You can't do that, string.empty is not a compile time constant, it's a static.